uniffi_bindgen/src/lib.rs

```diff
@@ -267,11 +280,8 @@ fn get_config(
     let config_file: Option<PathBuf> = match config_file_override {
         Some(cfg) => Some(PathBuf::from(cfg)),
         None => {
-            let path_guess = idl_file
-                .parent()
-                .ok_or_else(|| anyhow!("IDL file has no parent folder!"))?
-                .join("uniffi.toml");
```

Comment:
We need to `parent()` twice here, once for the filename and once for the `src` directory.